Makefile

```diff
@@ -2,7 +2,7 @@
 .PHONY:	requirements test test-requirements .tox
 
 uninstall:
-	while pip uninstall -y edx.analytics.tasks; do true; done
+	-for i in $$(seq 1 3); do pip uninstall -y edx.analytics.tasks; done
 	python setup.py clean
```

Comment:
@swalladge  Do you mind to try out this change, which I think is what fixed it upstream?

https://github.com/edx/edx-analytics-pipeline/commit/ae52cacb2208866394fbd4e01b80f4acdcb17826

Reply:
Oh nice, I see what that does - the newer pip version exits with a non-zero code if the package to uninstall is not installed. :+1: Much neater fix.

Reply:
Yeah.. hopefully the older pip doesn't cause any other problems with the requirements :)

Could you also add that new `requirements/pip.txt` file to the `requirements` make target, to [match upstream master](https://github.com/edx/edx-analytics-pipeline/blob/master/Makefile#L36-L39)?